style.css

```diff
@@ -0,0 +1,77 @@
+/* @media all and (max-width: 500px) { */
+* {
+  font-family: sans-serif;
+  padding-left: 1%;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: italic;
+  font-weight: 700;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.introName {
+  margin-top: 3rem;
+  height: 146px;
+  width: 80%;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: left;
+}
+
+.intro {
+  width: 327px;
+  height: 150px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  text-align: left;
+}
+
+.connect {
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.topHeader {
+  width: 327px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.media_links {
+  display: flex;
+  justify-content: flex-start;
+  gap: 20px;
+  list-style: none;
+  margin-top: -1rem;
+  font-size: 24px;
+}
```

Comment:
[OPTIONAL] Kindly check and update the colors of the media links to match the design since they are not the same currently. :wink:
 

Your design

![Screenshot from 2022-12-02 08-23-58](https://user-images.githubusercontent.com/65233860/205250490-8171fb3a-ff2c-4fe5-b64f-2de8453bc4bb.png)


The figma design

![Screenshot from 2022-12-02 08-23-06](https://user-images.githubusercontent.com/65233860/205250592-f2c8e53e-0e78-496e-80d5-b5a43ed6e218.png)

Reply:
@jerryowusu Hello Jerry,
I am trying to fix or at least understand what is the mistake in my mobile portfolio. The wight and height are according to the template for the mobile version. If I put the width on 100% will be responsive for the desktop version also but there will be a change of design required. 
I spoke with a couple of students and they used the same way of coding.

Please help me solve this problem before I submit my extra request for this module.

Reply:
@jerryowusu Hi again Jerry,
I have gone through the requirements and I didn't find that the implementation should be responsive for both desktop and mobile resolutions. It is only about mobile resolutions at the moment.
So, going back to my first reviewer, I have fixed the container that was going outside of the background and now the implementation is according to the design for a mobile version.

<img width="578" alt="Required Changes 1th review" src="https://user-images.githubusercontent.com/116429687/205271784-47a54f79-c25e-475e-8998-4d432d571ea8.png">